lidar_localizer/nodes/ndt_matching/ndt_matching.cpp

```diff
@@ -1560,31 +1547,45 @@ int main(int argc, char** argv)
   private_nh.getParam("imu_upside_down", _imu_upside_down);
   private_nh.getParam("imu_topic", _imu_topic);
   private_nh.param<double>("gnss_reinit_fitness", _gnss_reinit_fitness, 500.0);
+  private_nh.getParam("output_tf_frame_id", _output_tf_frame_id);
```

Comment:
The default value set in `_output_tf_frame_id` will be overwritten by this call. If the param doesn't exist, this will leave it with a blank string. Please provide a default value in this call.